lib/compiler.js

```diff
@@ -268,6 +285,9 @@ Compiler.prototype.visitAttributes = function(attrs) {
         }
       } else if (attr.name === `id` && attr.val[0] === `'`) {
         id = attr.val;
+      } else if (attr.name === `attrs`) {
```

Comment:
```suggestion
      } else if (attr.name === `attrs`) {
        // special case for snabbdom's 'attrs' object notation for specifying HTML attributes
        // since attributes are always strings, automatically JSON-serialize any object values
        // e.g.: in attrs={foo: {bar: 'baz'}} the {bar: 'baz'} should be auto-serialized
```